pandas/_testing/asserters.py

```diff
@@ -609,6 +609,43 @@ def assert_interval_array_equal(
     assert_attr_equal("inclusive", left, right, obj=obj)
 
 
+def assert_interval_array_almost_equal(
+    left, right, rtol, atol, obj="IntervalArray"
+) -> None:
+    """
+    Test that two IntervalArrays are within tolerance.
+
+    Parameters
+    ----------
+    left, right : IntervalArray
+        The IntervalArrays to compare.
+    rtol : float, default 1e-5
+        Relative tolerance.
+
+        .. versionadded:: 1.1.0
+    atol : float, default 1e-8
+        Absolute tolerance.
+
+        .. versionadded:: 1.1.0
+    obj : str, default 'IntervalArray'
+        Specify object name being compared, internally used to show appropriate
+        assertion message
+    """
+    _check_isinstance(left, right, IntervalArray)
+
+    kwargs = {}
+    if left._left.dtype.kind in ["m", "M"]:
```

Comment:
This is not hit by tests

Reply:
Not sure what to do. Should I take it out or make more robust tests?

Reply:
Is this check relevant? Then yes, you should add tests.

Reply:
Got it

Reply:
How can I test this line gets called since kwargs is created in the function and passed down to another function? I kept this in cause it's also in assert_interval_array_equal above this function. 
Should I just include a test for this function with a datetime/ timedelta array?

Reply:
I took out the kwargs entirely. assert_interval_array_almost_equal() calls assert_almost_equal() which calls pandas._libs.testing.assert_almost_equal() in pandas/_libs/testing.pyx. That function doesn't take in kwargs from the preceding functions. Since the kwargs (check_freq) is not needed, I took it out.